src/components/Table/Table.stories.js

```diff
@@ -0,0 +1,139 @@
+import Table from './Table.vue';
+import TableHeader from './TableHeader.vue';
+import TableBody from './TableBody.vue';
+import TableRow from './TableRow.vue';
+import Button from '../Button/Button.vue';
+import mdx from './Table.mdx';
+
+export default {
+  title: 'Components/Table',
+  component: Table,
+  subcomponents: { TableHeader, TableRow, TableBody, Button },
+  parameters: {
+    docs: {
+      page: mdx
+    }
+  },
+  argTypes: {
+    space: {
+      options: ['sm', 'md', 'lg'],
+      control: {
+        type: 'select'
+      }
+    }
+  }
+};
+
+const SimpleTableTemplate = (args, { argTypes }) => ({
+  props: Object.keys(argTypes),
+  components: { Table, TableHeader, TableRow, TableBody },
+  setup: () => ({ args }),
+  template: `
+  <div style="width: 700px">
```

Comment:
Can we put this wrapper div in a decorator for all the stories in this file? ([here's an example](https://github.com/lob/ui-components/blob/main/src/components/Dropdown/Dropdown.stories.js#L8))